qtpy/tests/test_qtgui.py

```diff
@@ -123,3 +123,26 @@ def test_qtextcursor_moveposition():
     assert cursor.position() == cursor.anchor()
     assert cursor.movePosition(QtGui.QTextCursor.NextWord, QtGui.QTextCursor.KeepAnchor, 3)
     assert cursor.selectedText() == "foo bar baz"
+
+
+def test_opengl_imports():
+    """Test for presence of QOpenGL* classes
+    These classes were members of QtGui in Qt5, but moved to QtOpenGL in Qt6.
+    QtPy makes them available in QtGui to maintain compatibility
```

Comment:
```suggestion
    """
    Test for presence of QOpenGL* classes.

    These classes were members of QtGui in Qt5, but moved to QtOpenGL in Qt6.
    QtPy makes them available in QtGui to maintain compatibility.
```

Format the docstring properly (per PEP 257) and consistently